lib/jwt_signed_request.rb

```diff
@@ -77,7 +77,11 @@ def self.request_body(request:)
   private_class_method :request_body
 
   def self.verified_headers?(request:, claims:)
-    parsed_headers = JSON.parse(claims['headers'])
+    parsed_headers = begin
+      JSON.parse(claims['headers'])
+    rescue JSON::ParserError
+      {}
```

Comment:
After some offline discussion I'm happy to proceed as-is. I'm happy to accept this is not a regression, though I'd like to discuss whether we should allow empty/no headers in a separate issue.